Libraries/LibWeb/Painting/DisplayListPlayerSkia.cpp

```diff
@@ -35,10 +35,13 @@ namespace Web::Painting {
 DisplayListPlayerSkia::DisplayListPlayerSkia(RefPtr<Gfx::SkiaBackendContext> context)
     : m_context(context)
 {
+    if (!m_context)
+        dbgln("Falling back to CPU Backend painter");
 }
 
 DisplayListPlayerSkia::DisplayListPlayerSkia()
 {
+    dbgln("Falling back to CPU Backend painter");
```

Comment:
I am afraid this one will result in lots of spam logging on pages with SVG images, since we always fallback to CPU rendering for them.

Reply:
Off hand do you have any reference site that could show this SVG? Just so I can see how bad it could be?

But I agree, you will also get a log each time a tab opens, so gets a little tiring for someone that it doing a lot of work.